src/schemas.ts

```diff
@@ -145,7 +145,6 @@ export const getByCategories = {
     properties: {
       categoryId: {
         type: 'array',
-        items: { $ref: 'http://graasp.org/#/definitions/uuid' },
```

Comment:
What if you add this back? I know there's some issues with this, so I'm just suggesting. If it doesn't work then maybe we want to manually check the ids format and throw from the endpoint.

Reply:
I test it with REGEX, it works now